torch/_refs/__init__.py

```diff
@@ -2139,6 +2142,61 @@ def flipud(a: TensorLikeType) -> TensorLikeType:
     return flip(a, (0,))
 
 
+def _repeat_if_defined(a: Optional[Tensor], sizes: int):
+    if a is not None:
+        return a.repeat(sizes)
+    return a
+
+
+@register_decomposition(torch.ops.aten.instance_norm)
```

Comment:
I think in this case, we don't actually need to register a decomposition since it's already a `CompositeImplicitAutograd` op (and so is automatically decomposed in the `__torch_dispatch__` level).

Reply:
Is there a test that complains when a decomposition for CompositeImplicitAutograd aten op is registered? Would this decomposition be used somewhere or ignored?

Reply:
@Chillee, could you please take a look at the above questions?

Reply:
I think it mostly won't be used/tested anywhere. I think there's a test that complains here: https://github.com/pytorch/pytorch/blob/master/test/test_ops.py#L1598

No big deal though imo.